infrastructure/variables.tf

```diff
@@ -57,9 +57,11 @@ variable "authorised-services" {
 }
 
 variable "document_management_valid_domain" {
-  default = "https://api-gateway.test.dm.reform.hmcts.net"
```

Comment:
Changing this is not required as long as it's using tactical. This should be reverted and left to use tactical `test`.